### PR TITLE
Pixel shifting out of the way port from Skyrat

### PR DESCRIPTION
**Pull request by NithaIsTired**
## About The Pull Request

Ports pixel shifting out of the way from skyrat.
https://github.com/Skyrat-SS13/Skyrat-tg/pull/15175

## Why It's Good For The Game

Makes it way more convenient to players that are pixel shifted won't get pushed around except from the direction they shifted in. This makes so those players that usually talking near a wall don't block the way or gets their RP interrupted.

## Changelog

:cl:
add: Allows pixel shifted players to not be pushed around depending on direction.
/:cl:

### CODE REVIEW

**Comment by thgvr** [comment]
Cool, this is something I was working on but got stuck. Have you tested this?

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -21,9 +21,9 @@
 		if(.) //not dead
 			handle_blood()
 
-		if(isLivingSSD())//if you're disconnected, you're going to sleep
-			if(AmountSleeping() < 20)
-				AdjustSleeping(20)//adjust every 10 seconds
+//		if(isLivingSSD())//if you're disconnected, you're going to sleep
+//			if(AmountSleeping() < 20)
+//				AdjustSleeping(20)//adjust every 10 seconds
```
Comment (thgvr) [inline code/modules/mob/living/carbon/life.dm]: what's up with this?

Comment (NithaIsTired) [reply code/modules/mob/living/carbon/life.dm]: argh I don't know why did it didn't revert that, it was just for testing

**Review by MarkSuckerberg — APPROVED** [review]
Oh, very cool

**Pull request merged**